platform/javascript/export/export.cpp

```diff
@@ -85,36 +85,44 @@ class EditorHTTPServer : public Reference {
 		// Wrong protocol
 		ERR_FAIL_COND_MSG(req[0] != "GET" || req[2] != "HTTP/1.1", "Invalid method or HTTP version.");
 
-		String filepath = EditorSettings::get_singleton()->get_cache_dir().plus_file("tmp_js_export");
+		const String cache_path = EditorSettings::get_singleton()->get_cache_dir();
 		const String basereq = "/tmp_js_export";
+		String filepath = "";
```

Comment:
For the reference `String` is value-initialized to `""` already so the assignment here is not necessary (but doesn't hurt either if it's for readability).